tumblr-downloader.go

```diff
@@ -150,67 +154,110 @@ func tumblrGetLikes(httpClient *http.Client, blogIdentifier string, timestamp in
 	return parser
 }
 
-func downloadURL(URL string) {
-	resp, err := http.Get(URL)
-	checkError(err)
+func downloadURL(URL string, SubDir string, AppendExtension string) {
+	fmt.Printf("Downloading %s ... ",URL)
```

Comment:
In a separate PR, we should also consider a uniform log format, and refactor these into a separate function.